tests/ascii_data_adiabatic_conditions.prm

```diff
@@ -0,0 +1,84 @@
+##### simple test for ascii data adiabatic conditions
+
+set Dimension                              = 2
+
+set Use years in output instead of seconds = true
+set End time                               = 1e6
+
+set Adiabatic surface temperature          = 1613.0
+
+subsection Adiabatic conditions model
+  set Model name = ascii data
+  subsection Ascii data model
+    set Data directory       = $ASPECT_SOURCE_DIR/data/adiabatic-conditions/ascii-data/
+    set Data file name       = simple_test.txt
```

Comment:
As mentioned above, I think it would be best to just move this file into the `tests/` directory, unless it is of any use to users.